pkg/asset/agent/installconfig.go

```diff
@@ -313,48 +333,16 @@ func warnUnusedConfig(installConfig *types.InstallConfig) {
 			fieldPath := field.NewPath("Platform", "Baremetal", "ExternalBridge")
 			logrus.Warnf(fmt.Sprintf("%s: %s is ignored", fieldPath, baremetal.ExternalBridge))
 		}
-		if baremetal.ProvisioningNetwork != defaultBM.ProvisioningNetwork {
-			fieldPath := field.NewPath("Platform", "Baremetal", "ProvisioningNetwork")
-			logrus.Warnf(fmt.Sprintf("%s: %s is ignored", fieldPath, baremetal.ProvisioningNetwork))
-		}
-		if baremetal.ProvisioningBridge != defaultBM.ProvisioningBridge {
-			fieldPath := field.NewPath("Platform", "Baremetal", "ProvisioningBridge")
-			logrus.Warnf(fmt.Sprintf("%s: %s is ignored", fieldPath, baremetal.ProvisioningBridge))
-		}
-		if baremetal.ProvisioningNetworkInterface != defaultBM.ProvisioningNetworkInterface {
-			fieldPath := field.NewPath("Platform", "Baremetal", "ProvisioningNetworkInterface")
-			logrus.Warnf(fmt.Sprintf("%s: %s is ignored", fieldPath, baremetal.ProvisioningNetworkInterface))
-		}
-		if baremetal.ProvisioningNetworkCIDR.String() != defaultBM.ProvisioningNetworkCIDR.String() {
-			fieldPath := field.NewPath("Platform", "Baremetal", "ProvisioningNetworkCIDR")
-			logrus.Warnf(fmt.Sprintf("%s: %s is ignored", fieldPath, baremetal.ProvisioningNetworkCIDR))
-		}
 		if baremetal.DeprecatedProvisioningDHCPExternal != defaultBM.DeprecatedProvisioningDHCPExternal {
```

Comment:
DeprecatedProvisioningDHCPExternal was replaced by setting the ProvisioningNetwork to "Unmanaged", so we should drop this warning also.

Reply:
Good point, removed.